.travis.yml

```diff
@@ -12,6 +12,18 @@ matrix:
         - python: "3.4"
         - python: "pypy3"
         - python: "pypy"
+before_install:
+    - identify -version
+    - convert -list format
+    - sudo apt-get remove imagemagick
+    - wget http://www.imagemagick.org/download/releases/ImageMagick-6.9.1-10.tar.xz
+    - tar xf ImageMagick-6.9.1-10.tar.xz
+    - cd ImageMagick*
+    - ./configure --disable-shared --with-openjp2=yes --with-webp=yes --prefix=/usr
+    - make
+    - sudo make install
```

Comment:
@guilhermef I'm using PIL now (pretty easy). But the nightmare continues. I have ImageMagick built on OSx with jpeg2000 support (see instructions: https://gist.github.com/okor/ed0bf4615109a2aedae8) but pillow does not seem to build properly on OSx.

```
Traceback (most recent call last):
  File "/Users/jason/Projects/voxmedia/thumbor-plugins/tests/test_optimizers.py", line 115, in test_jp2_should_optimize
    optimizer.optimize(None, fixtures_folder + '/img/fixture2.png', temp.name)
  File "/Users/jason/Projects/voxmedia/thumbor-plugins/thumbor_plugins/optimizers/jp2.py", line 34, in optimize
    return Image.open(input_file).save(output_file, 'JPEG2000', quality_mode=self.jp2_quality)
  File "/usr/local/lib/python2.7/site-packages/PIL/Image.py", line 1703, in save
    save_handler(self, fp, filename)
  File "/usr/local/lib/python2.7/site-packages/PIL/Jpeg2KImagePlugin.py", line 260, in _save
    ImageFile._save(im, fp, [('jpeg2k', (0, 0)+im.size, 0, kind)])
  File "/usr/local/lib/python2.7/site-packages/PIL/ImageFile.py", line 492, in _save
    e = Image._getencoder(im.mode, e, a, im.encoderconfig)
  File "/usr/local/lib/python2.7/site-packages/PIL/Image.py", line 437, in _getencoder
    raise IOError("encoder %s not available" % encoder_name)
IOError: encoder jpeg2k not available
```

After getting openjpeg installed (a version reported to work with both imagemagick and pillow - 2.0.1 and no later) I tried reinstalling pillow with `pip install -Iv pillow` and rerunnin the test but no luck. Any ideas?

As always, I'll keep digging. Any help appreciated.